i18n/en.json

```diff
@@ -1955,7 +1955,7 @@
   "help.formatting.headings2Example": "Large Heading\n-------------",
   "help.formatting.headingsExample": "## Large Heading\n### Smaller Heading\n#### Even Smaller Heading",
   "help.formatting.images": "## In-line Images\n\nCreate in-line images using an `!` followed by the alt text in square brackets and the link in normal brackets. Add hover text by placing it in quotes after the link.",
-  "help.formatting.imagesExample": "![alt text](link \"hover text\")\n\nand\n\n[![Build Status](https://travis-ci.org/mattermost/platform.svg?branch=master)](https://travis-ci.org/mattermost/platform) [![Github](https://assets-cdn.github.com/favicon.ico)](https://github.com/mattermost/platform)",
+  "help.formatting.imagesExample": "![alt text](link \"hover text\")\n\nand\n\n[![Build Status](https://travis-ci.org/mattermost/mattermost-server.svg?branch=master)](https://travis-ci.org/mattermost/mattermost-server) [![Github](https://assets-cdn.github.com/favicon.ico)](https://github.com/mattermost/mattermost-server)",
```

Comment:
Same for 
 - https://github.com/mattermost/mattermost-webapp/blob/master/components/help/components/formatting.jsx#L41
 - https://github.com/mattermost/mattermost-webapp/blob/master/components/help/components/formatting.jsx#L43

Reply:
Fixed 👍